### PR TITLE
chore: separate hanwritten and generated files in web-playground

### PR DESCRIPTION
**Pull request by pront**
closes: #17827 

I went with the simplest solution since we might do a major overhaul later. It should be good enough for now.

### CODE REVIEW

**Review by spencergilbert — APPROVED** [review]

File: lib/vector-vrl/web-playground/public/README.md [hunk 1]
```diff
@@ -65,7 +67,42 @@ but they will either error (enrichment functions) or abort (all the others) at r
 - `get_enrichment_table_record`
 
 Functions from VRL stdlib that are currently not supported can be found
-with this [issue filter][vrl-wasm-unsupported-filter]
+with this [issue filter][vrl-wasm-unsupported-filter].
+
+### macOS Troubleshooting
```
Comment (jszwedko) [inline lib/vector-vrl/web-playground/public/README.md:72]: Weren't these added in https://github.com/vectordotdev/vector/pull/17824 too? Maybe this was accidentally included here?

Comment (pront) [reply lib/vector-vrl/web-playground/public/README.md:72]: `wasm-pack build` essentially copies the `README.md` file into the generated package, in our case the `public` directory. Although we may not require `README.md` in the generated directory, the CLI args don't provide any way to overwrite this behavior.

Comment (pront) [reply lib/vector-vrl/web-playground/public/README.md:72]: This just made me realize I also need to update https://github.com/vectordotdev/vector/blob/master/lib/vector-vrl/web-playground/netlify.toml.

Comment (jszwedko) [reply lib/vector-vrl/web-playground/public/README.md:72]: Aha, I see. Should we be checking in the generated README?

Comment (pront) [reply lib/vector-vrl/web-playground/public/README.md]: No, it is unnecessary. And after my last commit we can also get rid of `public/.gitignore`.

**Review by jszwedko — APPROVED** [review]

**Pull request merged**